client/src/App.js

```diff
@@ -31,7 +31,8 @@ const Private = () => {
         <Route path="/payment" render={(props) => <Payment {...props} />} />
         <Route path="/test" component={TestComponent} />
         <Route path="/profile" component={CustomerProfile} />
-        <Route path="/chefprofile" component={ChefProfile} />
+        <Route path="/chefprofile/:userId?" component={ChefProfile} />
+        <Route path="/chef/:userId?/recipeId?" component={TestChefProfile} />
```

Comment:
```suggestion
        <Route path="/chef/:userId?/:recipeId?" component={TestChefProfile} />
```
I had to make this change to get the Chef profile to show, not sure why it worked before the recent changes

Reply:
This is happening because we had Login component on "/" route initially and it was redirecting us, since there is no component rending on "/", it's failing to redirect to proper component. I will have a discussion with the team and render a possible Home component on "/" route that will redirect to proper component based on the type of user logged in